index.md

```diff
@@ -111,4 +111,9 @@ Want to see nmstate in action? Check out the [demos](./demos.md).
 
 ## Stack support:
 - [IPv4 addresses (static)](./examples.md#interfaces-ethernet)
+- [IPv6 addresses (static)](./examples.md#interfaces-ethernet)
+    - Fliping IPv6 status(enable/disable) is not supported yet.
```

Comment:
We do not support setting IPv6 stack to disable. You can point to the Jira issue that tracks this.